core/src/main/java/org/infinispan/commands/functional/ManyDataCommand.java

```diff
@@ -0,0 +1,10 @@
+package org.infinispan.commands.functional;
+
+import java.util.Set;
+
+/**
+ * @author Radim Vansa &lt;rvansa@redhat.com&gt;
```

Comment:
It's all it is, just a superinterface for the functional commands that operate on multiple keys.